homework-submission/homework-week1/app.js

```diff
@@ -0,0 +1,74 @@
+const Item = ({ id, description, deadline, done }) => {
+  // a component has li element and the data which comes from another components(static and dynamic)
+  if (done == true) {
```

Comment:
A `true/false` are a little confusing for a developer who is trying to reach out the css files and figuring out what belongs to whom.
A css name like would be more convenient as a className by any case.

Another thing you may improve here, is the ternary operator as js1 remember? this is it's place to use!
so you will have:
```javascript
  <li className={done === true ? "crossed-line" : ""}>
    {id} {description} {deadline}
  </li>
```
and that's it no need for any if statement blocks!